src/main/java/org/folio/processor/rule/DataSource.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.processor.rule;
+
+import org.folio.processor.translations.Translation;
+
+public class DataSource {
```

Comment:
I am not sure, but I think we have to add extended documentation to all classes in the library

Reply:
Yes you are right, I'm going to push documentation not for all, but for most of the classes